tests/_support/Entity/User.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Tests\Support\Entity;
+
+use CodeIgniter\Entity;
+
+class User extends Entity
+{
+    protected $datamap = [];
+    protected $dates   = ['created_at', 'expiry'];
+    protected $casts   = [];
+}
```

Comment:
`CodeIgniter\Entity` is deprecated.

```suggestion
use CodeIgniter\Entity\Entity;

class User extends Entity
{
    protected $attributes = [
        'country' => 'India',
    ];
}
```